File: 2019-10-07/p1-factorial.py
from jutge import read

def factorial(n):
    """retorna n!, suposant n>=0 i n enter."""
    if n == 0:
        return 1
    else:
        return n * factorial(n - 1)

def main():
    n = read(int)
    print(factorial(n))

main()

File: 2019-10-07/p1-factorial.cc
#include <iostream>
using namespace std;

// retorna n!, suposant n>=0.
int factorial(int n) {
    if (n == 0) return 1;
    else return n * factorial(n - 1);
}

int main() {
    int n;
    cin >> n;
    cout << factorial(n) << endl;
}
